src/components/SettlementWindowsTab.js

```diff
@@ -317,6 +317,7 @@ function SettlementWindowsGrid(props) {
                   HUB 312 Report
                 </Button>
               </Grid>
+              /*
```

Comment:
Please add a comment as to why this is being kept.

If there is no reason to do so, then alternatively remove it please.

Reply:
Added comment.

644 report is no longer be used, more details can be found in the MES 1 Release notes https://modusbox.atlassian.net/wiki/spaces/CV/pages/2007334970/MES+1+Mowali+Release+Notes